DemoApp/ios/DemoApp/AppDelegate.h

```diff
@@ -7,9 +7,11 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  */
 
+#import <React/RCTBridgeDelegate.h>
 #import <UIKit/UIKit.h>
 
 @interface AppDelegate : UIResponder <UIApplicationDelegate>
+@interface AppDelegate : UIResponder <UIApplicationDelegate, RCTBridgeDelegate>
```

Comment:
Why importing `RCTBridgeDelegate` even though we're not using it? Is it added automatically by `react-native upgrade` or some script?

Reply:
I leaned on these changes https://github.com/react-native-community/rn-diff-purge/compare/version/0.58.6...version/0.59.0. I'm not sure, but It seems to be necessary

Reply:
Please do the same for other apps

Reply:
@dtrabo okay, that's already.